spec/helpers/categories_helper_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
```

Comment:
all of the methods you've defined within your `CategoriesHelper` should be tested here 👍